Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies.cs

```diff
@@ -258,22 +346,22 @@ private async Task NavigateToNieuwsWerkinstructiesBeheer()
 
         await Expect(beheerlink).ToBeVisibleAsync(new() { Visible = false });
 
-        if (await berichtenlink.GetAttributeAsync("aria-current") != "page")
+        if (await skillslink.GetAttributeAsync("aria-current") != "page")
         {
-            await berichtenlink.ClickAsync();
+            await skillslink.ClickAsync();
         }
     }
 
-    private async Task CreateBericht(string titel, bool isBelangrijk)
+    private async Task CreateBericht(string titel, bool isBelangrijk, string skill, bool changePublishDate = false)
```

Comment:
van changepublishdate is nogal onduidelijk wat deze gaat doen. duidelijker als je de gewenste waarde, of de offset meegeeft.

Reply:
aangepast